imap_processing/tests/test_decom.py

```diff
@@ -0,0 +1,61 @@
+"""Tests aspects of the decommutation process that is generic to all instruments"""
+
+import glob
+
+import pytest
+
+from imap_processing import imap_module_directory
+
+xtce_documents = glob.glob(f"{imap_module_directory}/*/packet_definitions/*.xml")
+xtce_documents = [
+    item for item in xtce_documents if "idex_packet_definition.xml" not in item
+]  # Temporary until IDEX packet definition is fixed
+
+
+@pytest.mark.parametrize("xtce_document", xtce_documents)
+def test_ccsds_header(xtce_document):
+    """Test if the XTCE document contains the proper CCSDS header information"""
+
+    print(f"\nTesting {xtce_document}")
```

Comment:
```suggestion
```
This is already in the really long parameterization name of the tests.
`imap_processing/tests/test_decom.py::test_ccsds_header[/home/runner/work/imap_processing/imap_processing/imap_processing/ultra/packet_definitions/P_U45_IMAGE_RATES.xml]`
You might be able to trim that somehow to make a nicer `id=filename` somehow, but not sure how much effort that would be.

Reply:
Probably not really necessary to have anything printed here. It was just handy while I was developing this.